src/components/index.js

```diff
@@ -0,0 +1,10 @@
+import Feature from "./Feature/Feature";
+import Footer from "./Footer/Footer";
+import Home from "./Home/Home";
```

Comment:
there is an issue here you exporting with working naming 
`import Home from './home/home';
`